README.md

```diff
@@ -210,14 +209,14 @@ import value from "module" if { type: "json" };
 
```

Comment:
I think we should clarify why we're pushing for `if` and the extra level of nesting in this section.